cdf-core/cdf/js-modules/dashboard/Utils.js

```diff
@@ -52,14 +53,51 @@ define(['../lib/underscore', '../lib/jquery', '../lib/queryParser'], function(_,
     return window.location.search;
   };
 
-  Utils.urlParams = undefined;
+  var urlParams = undefined;
 
   Utils.getQueryParameter = function(parameterName) {
-    if(Utils.urlParams === undefined) {
-      Utils.urlParams = $.parseQuery(this.getLocationSearchString());
+    if(urlParams === undefined) {
```

Comment:
made this change so that matches what is on master branch. 'urlParams' should be a private variable.